app/src/main/java/com/taskruler/MainActivity.kt

```diff
@@ -19,7 +19,7 @@ import org.koin.androidx.viewmodel.ext.android.viewModel
 
 class MainActivity : ComponentActivity() {
 
-    private val viewModel: MainViewModel by viewModel<MainViewModel>()
+    private val viewModel: MainViewModel by viewModel()
```

Comment:
This same suggestion appeared in a previously submitted code review.
When that occurs, only the first gets credit.

Reply:
This was a change I made in the previous code review, I did it for the same group so I used the same branch, is it possible you're looking at the previous pull request? The last 3 changes listed were from code review 1 but only got responses today. I am unsure of why these are all listed together but they are outdated. Please let me know if there is something I can do to straighten this out.